tests/unit/task_system_test.c

```diff
@@ -133,16 +130,25 @@ CTEST_SETUP(task_system)
    uint64 all_threads_inactive_mask = (~0L);
    uint64 this_thread_active_mask   = (~0x1L);
    uint64 exp_bitmask = (all_threads_inactive_mask & this_thread_active_mask);
-   ASSERT_EQUAL(exp_bitmask, task_bitmask);
+
+   ASSERT_EQUAL(exp_bitmask,
+                task_bitmask,
+                "exp_bitmask=0x%x, task_bitmask=0x%x\n",
+                exp_bitmask,
+                task_bitmask);
 
    // Save it off, so it can be used for verification in a test case.
    data->active_threads_bitmask = exp_bitmask;
+   if (Ctest_verbose) {
+      platform_set_log_streams(stdout, stderr);
+   }
```

Comment:
Thanks for adding this here.  In the other unit test files that have it, I've put it at the top of `CTEST_SETUP()`, in part so that the users's choice for "verbose" is respected even for test-setup outputs.

Reply:
Done.